python/pyspark/worker.py

```diff
@@ -25,7 +25,11 @@
 # 'resource' is a Unix specific module.
 has_resource_module = True
 try:
-    import resource
+    from resource import RLIMIT_AS
+    from resource import RLIM_INFINITY
+    from resource import getrlimit
+    from resource import setrlimit
+    from resource import error
```

Comment:
Oh, I guess you faced an issue about relative/absolute import issue? it will get confused in Python 2. If you're under `pyspark` module `resource` module can refer both `pyspark.resource` and `resource` (Python standard library). 

To avoid this, you can do `from __future__ import absolute_import` for now. We can remove it out when we drop Python 2, which will happen in Spark 3.1 anyway though.

Reply:
yes unfortunately now have a name conflict.  thanks, this is what I needed.